src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MethodContextReuseConstraints.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal struct MethodContextReuseConstraints
+    {
+        private readonly int _methodToken;
+        private readonly int _methodVersion;
+        private readonly uint _startOffset;
+        private readonly uint _endOffsetInclusive;
```

Comment:
Yeah, I'm pretty sure they're symmetrical.  For every +1/-1 you remove, you add one in another location.